cgnsutilities/cgnsutilities.py

```diff
@@ -2288,9 +2314,18 @@ def addBocoDataSet(self, bocoDataSet):
 
     def coarsen(self, direction):
         """Coarsen the range of the BC along the specified direction"""
+
         for j in range(2):
             self.ptRange[direction, j] = (self.ptRange[direction, j] - 1) // 2 + 1
```

Comment:
Does this need to change like in the `b2b` function?

Reply:
yep, fixed in the new version